src/plugin/render_operations/renderer_update_operation.cpp

```diff
@@ -14,6 +14,7 @@ namespace wmr
 		: MHWRender::MUserRenderOperation(name)
 		, m_renderer(dynamic_cast<const ViewportRendererOverride*>(MHWRender::MRenderer::theRenderer()->findRenderOverride(settings::VIEWPORT_OVERRIDE_NAME))->GetRenderer())
 	{
+		LOG("Attempting to get a reference to the renderer.");
```

Comment:
It actually already tried to get a reference when this LOG is called 😉 Doesn't matter anyway

Reply:
👍